pkg/kubelet/rkt/rkt.go

```diff
@@ -876,7 +917,7 @@ func (r *Runtime) generateRunCommand(pod *api.Pod, uuid string) (string, error)
 // On success, it will return a string that represents name of the unit file
 // and the runtime pod.
 func (r *Runtime) preparePod(pod *api.Pod, pullSecrets []api.Secret) (string, *kubecontainer.Pod, error) {
-	// Generate the pod manifest from the pod spec.
+	// Generate the appc pod manifest from the k8 pod spec.
```

Comment:
ah, I thought the 8 was pronounced `ubernetes`, not `ubernete`. Thanks! :ferris_wheel: